app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    # cache stores in format [[data-1, result-1] ... [data-n, result-n]]
+    results = []
```

Comment:
You should use dict here

Reply:
Yeah, I forgot that tuples from *args can be as keys in dict. Thanx!